tools/devtool

```diff
@@ -822,6 +822,7 @@ mount_ramdisk() {
     umount_ramdisk
```

Comment:
I'd also add a check for the `umount` call in `umount_ramdisk`, which would require the same privileges as mount

Reply:
I added a check in `umount_ramdisk`!